scss/_settings_colors.scss

```diff
@@ -12,21 +13,20 @@ $color-mid: #999 !default;
 $color-mid-dark: #666 !default;
 $color-dark: #111 !default;
 $color-x-dark: #000 !default;
-
-$color-negative: #c7162b !default;
-$color-caution: #f99b11 !default;
-$color-positive: #0e8420 !default;
-$color-information: #335280 !default;
+$color-input-shadow: rgba($color-x-dark, 0.12) !default;
 
 $color-label-validated: #006b75;
 
-$color-input-shadow: rgba($color-x-dark, 0.12) !default;
-
-$color-accent: $color-brand !default;
-$color-accent-background: $color-accent !default;
+$color-negative: #c7162b !default;
+$color-caution: #f99b11 !default;
+$color-positive: #0e8620 !default;
 
-// for visual outline on :active / :focus elements
-$color-focus: #19b6ee !default;
+$color-link: #06c !default;
+$color-link-visited: #7d42b8 !default;
+$color-focus: #2e96ff !default;
+$color-focus-positive: #003008 !default;
+$color-focus-negative: #3b0006 !default;
+$color-information: #24598f !default;
```

Comment:
Could `$color-information` be moved back up with its "siblings" (positive, negative, caution)? I thought it was removed by mistake when it wasn't there.
We seem to have colors in logical groups in this file, so let's keep them this way for easier reference.